grails-app/domain/org/gmod/chado/CvtermRelationship.groovy

```diff
@@ -2,15 +2,15 @@ package org.gmod.chado
 
 class CvtermRelationship {
 
-    Cvterm cvtermBySubjectId
-    Cvterm cvtermByObjectId
-    Cvterm cvtermByTypeId
+    Cvterm subject
+    Cvterm object
+    Cvterm type
```

Comment:
While what I said is right, what you did is more correct.   subject as a foreign key uses "subject_id" by default to do the mapping so you should be good.   Just double-check the constraints (e.g., nullable).  That is slick!